app/services/feature_flag.rb

```diff
@@ -15,6 +15,7 @@ def feature
   PERMANENT_SETTINGS = [].freeze
 
   TEMPORARY_FEATURE_FLAGS = [
+    [:longer_than_normal, 'It’s taking us longer than usual to find TRNs', 'Felix Clack'],
```

Comment:
What do you think of this feature flag name which makes it a bit more generic? Also updated the description.

```suggestion
    [:processing_delays, 'Show users banners and interstitials warning them of increased waiting times', 'Felix Clack'],
```